flocker/__init__.py

```diff
@@ -11,27 +11,6 @@
 REST_API_PORT = 4523
 
 
-def _disable_pyrsistent_c_extensions():
-    """
-    Pyrsistent sometimes segfaults. Disabling the C extension reduces the
-    likelihood of this happening.
-
-    We do this first so it happens before pyrsistent is imported.
-
-    In theory this bug is fixed in pyrsistent 0.11.7, so this may no
-    longer be necessary and is likely worth risking for higher
-    performance. Once we have benchmarking framework we can assess
-    risk/benefit ratio better.
-
-    The mechanism for disabling extensions is documented at
-    https://github.com/tobgu/pyrsistent/blob/master/CHANGES.txt#L17-L19.
-    """
-    import os
-    os.environ[b"PYRSISTENT_NO_C_EXTENSION"] = b"1"
-_disable_pyrsistent_c_extensions()
-del _disable_pyrsistent_c_extensions
```

Comment:
:question: Do we still think this gave a significant speedup? ISTR it did, but not as much as other Python fixes. Perhaps we should disable it again to avoid the risk of segfaults in the field.

@itamarst or @exarkun the Pyrsistent changelog says segfaults were fixed in 0.11.7. Did we consciously keep the c extension disabled after we upgraded to 0.11.9?